src/session10/homework/bank_account_management/BankAccount.java

```diff
@@ -0,0 +1,48 @@
+package session10.homework.bank_account_management;
+
+import java.util.UUID;
+
+public class BankAccount {
+
+    String accountHolderName;
```

Comment:
Please reconsider having just one property for the account holder. Try to see pros and cons of this approch and maybe think of having 2 properties, one for each first name/last name.